test/algorithm/find_if.hpp

```diff
@@ -5,11 +5,14 @@
 #pragma once
 
 #include <type_traits>
+
 #include <kvasir/mpl/algorithm/find_if.hpp>
-#include <kvasir/mpl/functional/bind.hpp>
+
+template<typename T>
+using is_char = std::is_same<char, T>;
 
 using namespace kvasir;
-static_assert(std::is_same<mpl::find_if<mpl::bind<std::is_same, char>::template f,
+static_assert(std::is_same<mpl::find_if<is_char,
```

Comment:
this is why we should provide a bind1, that does work with continuations despite the alias flaw